server/.env

```diff
@@ -1 +1,4 @@
-TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam"
+TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Kenneth Gilmore"
+
+DB_URI="mongodb+srv://admin:kenneth123@cluster0.ts4jw.mongodb.net/messaging-app?retryWrites=true&w=majority"
+
```

Comment:
You might want to consider adding this to .gitignore. You should probably also remove the mention of the specific URI from your README as well.

Reply:
Yeah I agree this is bad practice and have removed from the readme. Just unsure how to allow users the ability to access database locally without URI as per the instructions.

Reply:
I'm not too sure but I _think_ we should be able to swap that URI out with a locally hosted database URI.